CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+# Code of Conduct
+
+## When something happens
+
+If you see a violation of our community's standards of [Conduct](#conduct), follow these steps:
+
+1. Let the person know that what they did is not appropriate and ask them to stop and/or edit their messages.
+2. That person should immediately stop the behavior and correct the issue.
+3. If this doesn’t happen, or if you're uncomfortable speaking up, contact the maintainers.
+4. As soon as available, a maintainer will look into the issue, and take further action, starting with a warning, then temporary block, then long-term repo or organization ban.
+
+When reporting, please include any relevant details, links, screenshots, context, or other information that may be used to better understand and resolve the situation.
+
+The maintainers will prioritize the well-being and comfort of the recipients of the violation over the comfort of the violator.
```

Comment:
This is of course for @cart do decide for now since he will be the only enforcer, but I copied it over anyway since I think it's a good statement.

Reply:
I've removed this part because it can always be added later if need be, for this initial draft the specific enforcement steps aren't necessary.